<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TenkeyDialog</class>
 <widget class="QDialog" name="TenkeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="one_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="two_button">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>150</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="three_button">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>150</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="six_button">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>230</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="five_button">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>230</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="four_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nine_button">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>310</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="eight_button">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>310</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="seven_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="zero_button">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>390</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backspace_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/pngtree-backspace-button-3d-icon-render-png-image_6174303-2377787671.png</normaloff>icons/pngtree-backspace-button-3d-icon-render-png-image_6174303-2377787671.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>86</width>
     <height>86</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="accept_button">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>390</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/accept.png</normaloff>icons/accept.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="enteredtext_lineedit">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>40</y>
     <width>304</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>108</pointsize>
    </font>
   </property>
   <property name="text">
    <string>99/99</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>390</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/cancel.png</normaloff>icons/cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>cancel_button</zorder>
  <zorder>one_button</zorder>
  <zorder>two_button</zorder>
  <zorder>three_button</zorder>
  <zorder>six_button</zorder>
  <zorder>five_button</zorder>
  <zorder>four_button</zorder>
  <zorder>nine_button</zorder>
  <zorder>eight_button</zorder>
  <zorder>seven_button</zorder>
  <zorder>zero_button</zorder>
  <zorder>backspace_button</zorder>
  <zorder>accept_button</zorder>
  <zorder>enteredtext_lineedit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
